cgmanifest.json

```diff
@@ -10981,18 +10981,8 @@
         "type": "other",
         "other": {
           "name": "librdkafka",
-          "version": "1.4.0",
-          "downloadUrl": "https://github.com/edenhill/librdkafka/archive/v1.4.0.tar.gz"
-        }
-      }
-    },
-    {
-      "component": {
-        "type": "other",
-        "other": {
-          "name": "librdkafka",
-          "version": "1.5.0",
```

Comment:
Please also remove `librdkafka` from `SPECS-EXTENDED`. It looks like we have an outdated duplicate.